dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1140,7 +1147,7 @@ public void launch(final List<String> args) {
      */
     public abstract String zipFilename(T entry);
 
-    private void launchCwl(String entry, final List<String> args, boolean isLocalEntry) throws ApiException, IOException {
+    private void launchCwl(String entry, final List<String> args, boolean isALocalEntry) throws ApiException, IOException {
```

Comment:
You have a method on the class `isLocalEntry()`, but are passing in a parameter here and not using the method. Is there a reason for having it both ways? (just skimmed the code so far, so I may be missing something).